dist/svg/ds4/icons.svg

```diff
@@ -344,4 +344,36 @@
     <symbol id="icon-back" viewBox="0 0 13 23">
         <path d="M12.118.263a.92.92 0 00-1.288 0L.293 10.8c-.176.176-.293.41-.293.644 0 .234.117.468.293.644l10.185 10.185c.176.176.41.293.644.293.234 0 .468-.117.644-.293a.92.92 0 000-1.288l-9.541-9.54 9.893-9.894a.92.92 0 000-1.288z"/>
     </symbol>
+<!--alias of icon-checkbox-checked for forwards compatibility-->
```

Comment:
Just curious ~

How did you generate these icons?

Reply:
https://github.com/eBay/skin/blob/master/ICON-CREATION.md